code/game/objects/structures/watercloset.dm

```diff
@@ -414,6 +414,16 @@
 		user << "<span class='warning'>Someone's already washing here!</span>"
 		return
 
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/monkeycube))
+		if (/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped)

```

Comment:
this seems a bit non-standard, especially in combination with the return /obj/item/weapon/reagent_containers/food/snacks/monkeycube/proc/Expand() below.

Usuallllyyy for stuff like this, you'd typecast to monkeycubes, then wreck for if they're wrapped or not (after all, maybe other monkey cubes, in the future, may be wrapped but not have this path), then you could also do x.expact() later on down.